.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+
+# One line description
+
+Please include a paragraph here describing what this PR does:
+what kind of change (refactoring, bugfix, feature), what is changed
+(short summary of new features or fix).
+
+If bugs are addressed in this PR, include the issue numbers.
+
```

Comment:
I'd suggest adding an extra section to explicitly call out new/modified tests and examples.

Reply:
I don't think there is a need to call each modified test and example here. The Git history of the branch already captures this well. This adds unnecessary overhead to our process.

Reply:
If a developer (maybe @kennyweiss) feels it will make his PR document more useful, I think he should list the tests and examples.  If a developer (perhaps @gzagaris) feels something in the PR template is unnecessary burden, that developer should delete that part of the filled-in text when creating the PR.  Then the reviewers can either ignore the list of tests and examples or ask for what they feel is the missing information.

Reply:
I don't think we need an explicit "tests" section, but the developer could add it.